src/main.c

```diff
@@ -26,6 +30,18 @@ void blinkyTask(void *dummy){
 	}
 }
 
+void ADCTask(void *dummy){
```

Comment:
It would be better to move this task from the main.c file to something like adc.c